src/Command/ShowCommand.php

```diff
@@ -216,7 +219,7 @@ private function writeTraceCodeSnippet(OutputInterface $output, array $trace, $i
             return;
         }
 
-        $output->write($this->getHighlighter()->getCodeSnippet($code, $line, 5, 5), ShellOutput::OUTPUT_RAW);
+        $output->write($this->getHighlighter()->getCodeSnippet($code, $line, 5, 5), false, OutputInterface::OUTPUT_RAW);
```

Comment:
This won't work on Symfony 3.

Reply:
It will : the argument is present since the very first commit of the interface, which was introduced in august 2010.

Reply:
(So this means this was faulty to begin with...)

Reply:
See https://github.com/symfony/console/commit/68c191cd02f0dbffeef342e95c57cf741008abfd

Reply:
Found the real commit which introduced this parameter : https://github.com/symfony/symfony/commit/e715bc3d45496e77d52b497ef65ef98aa1e99906

One question pops to mind though : will we want to add a new line at the end or not ? As if the `ShellOutput::OUTPUT_RAW` (which is `OutputInterface::OUTPUT_RAW` : it is imo less misleading to use the proper class for the constants, especially if they aren't overwritten - I'm not even sure it's actually possible) was `1`, then maybe I should use `true` instead to avoid a BC break ?

WDYT @bobthecow, @GrahamCampbell ?

Reply:
In this particular case, it doesn’t make a difference. The output is the same either way. I’d probably lean toward `false`.